_scripts/instruction-widget/install.js

```diff
@@ -194,6 +199,8 @@ module.exports = function(context) {
     context.dns_package_prefix = "certbot-dns";
   }
 
+  // This function is currently unused, but we keep it around to make it easy
```

Comment:
Have you a specific short/middle-term use case in mind to keep the function around? Otherwise I think that the git history may be sufficient here.

Reply:
I think if we find a significant problem with our snaps on these systems, we'd want to change things back.

Another case is we want instructions for other architectures as suggested [here](https://github.com/certbot/website/issues/619#issuecomment-668205864).

I could clean some of this dead code up, but this PR follows the same pattern as https://github.com/certbot/website/pull/627 and https://github.com/certbot/website/pull/648 and I'd personally like to continue it and we can clean everything up later when we know we don't need it.

Reply:
Ah I just noticed I did clean up the unused branches in those PRs so I'm happy to do that here. I'd still like to leave this function around though.

Reply:
That works for me. With the branch removed, it becomes clear from any decent IDE that the function is unused and can be deleted during future refactorings.